src/platform-implementation-js/dom-driver/gmail/gmail-element-getter.ts

```diff
@@ -175,14 +172,6 @@ const GmailElementGetter = {
 
     // leftNavElement classnames depending on gmail chat & meet settings:
 
-    // Old Hangouts-only Gmail
-    //  div.aeN.WR.nH.oy8Mbf.nn
-    //  (looks identical to current Gmail's chat-off-meet-off setup, but there are important
-    //  CSS differences inside the element!)
-    // ===================
-    // No classname difference between open/collapsed
-    //  div.ajl.aib.aZ6 (classic hangouts on left or right, google meet on/off)
```

Comment:
https://support.google.com/chat/answer/9341104?hl=en&co=GENIE.Platform%3DDesktop suggests otherwise, but chat position as a setting seems to not exist as it did in hangouts. 

<img width="1624" alt="Screenshot 2023-08-16 at 13 28 58" src="https://github.com/InboxSDK/InboxSDK/assets/5156873/3e66c7d5-d32b-4d01-bb86-3ce5cd35e4d7">